### PR TITLE
convert to new override syntax

### PR DESCRIPTION
**Pull request by MarkusVolk**
Signed-off-by: Markus Volk <f_l_k@t-online.de>

### CODE REVIEW

File: recipes-graphics/waylandpp/waylandpp_0.2.8.bb [hunk 1]
```diff
@@ -35,19 +35,19 @@ EXTRA_OECMAKE_append_class-target = " \
...
+FILES:${PN}-dev  = " \
     ${includedir}/* \
     ${libdir}/pkgconfig/* \
     ${libdir}/cmake/waylandpp/* \
     ${datadir}/waylandpp/protocols/* \
     "
 
-FILES_SOLIBSDEV = ""
-INSANE_SKIP_${PN} += "dev-so"
+FILES:SOLIBSDEV = ""
```
Comment (dev-0x7C6) [inline recipes-graphics/waylandpp/waylandpp_0.2.8.bb]: Not sure there, I think `FILES_SOLIBSDEV` is just a variable.

Comment (MarkusVolk) [reply recipes-graphics/waylandpp/waylandpp_0.2.8.bb]: Also thought about that. To me SOLIBSDEV looks like a package-name, but i'm unsure too. I am even unsure if that line is needed at all

**Comment by MarkusVolk** [comment]
You are right. Its just a variable. Fixed it and also one missed conversion in x265.bb

**Comment by dev-0x7C6** [comment]
Looks good, thanks :)

**Comment by dev-0x7C6** [comment]
Changes made to master and hardknott branch.

**Pull request merged**